search/2015-02-28-Preview/swagger/searchservice.json

```diff
@@ -844,114 +947,1820 @@
+      ],
+      "x-ms-enum": {
+        "name": "MicrosoftStemmingTokenizerLanguage",
+        "modelAsString": false
+      },
+      "description": "Lists the languages supported by the Microsoft language stemming tokenizer."
+    },
+    "MicrosoftLanguageStemmingTokenizer": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.MicrosoftLanguageStemmingTokenizer",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Tokenizer"
+        }
+      ],
+      "properties": {
+        "maxTokenLength": {
+          "type": "integer",
+          "format": "int32",
+          "default": 255,
+          "description": "The maximum token length. Tokens longer than the maximum length are split. Maximum token length that can be used is 300 characters. Tokens longer than 300 characters are first split into tokens of length 300 and then each of those tokens is split based on the max token length set. Default is 255."
+        },
+        "isSearchTokenizer": {
+          "type": "boolean",
+          "default": false,
+          "description": "A value indicating how the tokenizer is used. Set to true if used as the search tokenizer, set to false if used as the indexing tokenizer. Default is false."
+        },
+        "language": {
+          "$ref": "#/definitions/MicrosoftStemmingTokenizerLanguage",
+          "description": "The language to use."
+        }
+      },
+      "description": "Divides text using language-specific rules and reduces words to their base forms."
+    },
+    "NGramTokenizer": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.NGramTokenizer",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Tokenizer"
+        }
+      ],
+      "properties": {
+        "minGram": {
+          "type": "integer",
+          "format": "int32",
+          "default": 1,
+          "description": "The minimum n-gram length. Default is 1."
+        },
+        "maxGram": {
+          "type": "integer",
+          "format": "int32",
+          "default": 2,
+          "description": "The maximum n-gram length. Default is 2."
+        },
+        "tokenChars": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TokenCharacterKind"
+          },
+          "description": "Character classes to keep in the tokens."
+        }
+      },
+      "description": "Tokenizes the input into n-grams of the given size(s). This tokenizer is implemented using Apache Lucene.",
+      "externalDocs": {
+        "url": "http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/ngram/NGramTokenizer.html"
+      },
+      "x-ms-external": true
+    },
+    "PathHierarchyTokenizer": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.PathHierarchyTokenizer",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Tokenizer"
+        }
+      ],
+      "properties": {
+        "delimiter": {
+          "type": "string",
+          "default": "/",
```

Comment:
Good catch. We'll upgrade AutoRest after we ship 2.x-preview, but in the meantime I think I forgot to modify the generated code by hand...